homeassistant/components/ovo_energy/__init__.py

```diff
@@ -28,6 +30,13 @@
 PLATFORMS = [Platform.SENSOR]
 
 
+class OVOCoordinatorData(BaseModel):
```

Comment:
Is there really a need to introduce pydantic model for this instead of using a namedtuple or a dataclass?

Reply:
Yeah we should not use a Pydantic model for this.

Reply:
Removed